appeals/web/src/server/appeals/appeal-details/site-visit/site-visit.mapper.js

```diff
@@ -541,10 +541,7 @@ export function mapPostScheduleOrManageSiteVisitCommonParameters(
  * @param {import('./site-visit.service.js').UpdateOrCreateSiteVisitParameters} updateOrCreateSiteVisitParameters
  * @returns {ScheduleOrManageSiteVisitConfirmationPageType}
  */
-export function mapPostScheduleOrManageSiteVisitConfirmationPageType(
-	appealDetails,
-	updateOrCreateSiteVisitParameters
-) {
+export function getSiteVisitChangeType(appealDetails, updateOrCreateSiteVisitParameters) {
```

Comment:
Was there a reason to change this other than shortening it? I don't think this name is as descriptive as the one it replaces

Reply:
reason for change is that the function is no longer being used solely to render the page, it is now also passed to the api to make decisions on notify comms